packages/shared/src/components/cards/SharedPostCardFooter.tsx

```diff
@@ -2,39 +2,51 @@ import classNames from 'classnames';
 import React, { ReactElement } from 'react';
 import { Post } from '../../graphql/posts';
 import { cloudinary } from '../../lib/image';
+import { IconSize } from '../Icon';
+import { CardImage, CardVideoImage } from './Card';
 
 type SharedPostCardFooterProps = {
   isShort: boolean;
+  isVideoType?: boolean;
 } & Pick<Post, 'sharedPost'>;
 
 export const SharedPostCardFooter = ({
   sharedPost,
   isShort,
+  isVideoType,
 }: SharedPostCardFooterProps): ReactElement => {
+  const ImageComponent = isVideoType ? CardVideoImage : CardImage;
   return (
     <div
       className={classNames(
-        'flex flex-1 gap-2 p-3 mb-2 rounded-12 border border-theme-divider-tertiary',
```

Comment:
Should we be able to use the defined tailwind classes for min or max height? I wasn't able to, thus max-h-[13.5rem] & min-h-[0]

Reply:
What did you try to achieve, missing the context a bit.
Feel you can probably use defaults yes.

Reply:
I tried deleting .next and all possible things for this, but didn't work, let me try one more time 🤷‍♀️

Reply:
removed usage of `max-h-[13.5rem]` as it was no longer needed.